models/workitemtype_repository.go

```diff
@@ -11,6 +11,8 @@ import (
 	"github.com/jinzhu/gorm"
 )
 
+var cache = NewWorkItemTypeCache()
```

Comment:
I think the code would be clearer if we passed the cache as a dependency to the WorkItemTypeRepository. Using a global var makes it quite unclear who uses the cache where and when.